.travis.yml

```diff
@@ -33,8 +34,9 @@ script:
   - python3 -m flake8 --exclude=build,doc,example,tmp,amici_models
   - travis_wait 20 python3 -m pytest --cov=pypesto ./test/test_*
   - travis_wait 20 xvfb-run -a python3 -m pytest --cov=pypesto --cov-append ./test/visualize/test_*
-  - coverage xml
+  - if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then sphinx-build -W -b html -d doc/_build/html doc/ doc/_build/html; fi
```

Comment:
Can that go into the sphinx Makefile?

Reply:
why?

Reply:
because `make html` is way more readable

Reply:
yeah but I would leave the default options in make, s.t. the docs do not error upon minor inconsistencies (which are turned into errors here via -W). but feel free to edit it, possibly with options.

Reply:
`make travis` `make rtd` ... ?